### DISCUSSION TITLE
002_github_blog/003_naver_search_advisor

### DISCUSSION THREAD

**Discussion by giscus** [boyinblue/boyinblue.github.io#15] [General]
# 002_github_blog/003_naver_search_advisor

네이버 서치 어드바이저에 GitHub 블로그 등록 방법에 대해서 설명합니다.

https://boyinblue.github.io/002_github_blog/003_naver_search_advisor.html

**Comment by boyinblue** [comment]
개별로 댓글을 남길 수 있는가? 
즉, 이 페이지에 남긴 댓글이 다른 페이지에도 표시가 되는가?
